lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const types = require('../utils/types');
+
+const ERROR_MESSAGE = 'Use `\|\|` or the ternary operator instead of `getWithDefault()`'; // eslint-disable-line no-useless-escape, prettier/prettier
+
+module.exports = {
+  ERROR_MESSAGE,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Use the `\|\|` operator instead of `getWithDefault` for more expected behaviors', // eslint-disable-line no-useless-escape, prettier/prettier
```

Comment:
It looks like the README issue has not been fixed. Actually, can you just change the description to:

```
Disallows use of the Ember's `getWithDefault` function
```